setup.py

```diff
@@ -129,12 +130,7 @@ def build_extensions(self):
         if cython:
             self.render_templates(_pxifiles)
 
-        numpy_incl = pkg_resources.resource_filename("numpy", "core/include")
-
-        for ext in self.extensions:
-            if hasattr(ext, "include_dirs") and numpy_incl not in ext.include_dirs:
-                ext.include_dirs.append(numpy_incl)
-        _build_ext.build_extensions(self)
+        super().build_extensions()
```

Comment:
Can you explain why this is drop-able?

Reply:
If you take a look at the OP you'll see we at some point in the past just basically copy / pasted what we inherited from. So instead of doing that a super call gets us where we need to be with less code / maintenance